bindings/wasm/src/account/storage/traits.rs

```diff
@@ -43,52 +43,44 @@ extern "C" {
   pub type PromiseSignature;
   #[wasm_bindgen(typescript_type = "Promise<boolean>")]
   pub type PromiseBool;
-  #[wasm_bindgen(typescript_type = "Promise<Generation>")]
-  pub type PromiseOptionGeneration;
-  #[wasm_bindgen(typescript_type = "Promise<ChainState>")]
+  #[wasm_bindgen(typescript_type = "Promise<ChainState | undefined | null>")]
   pub type PromiseOptionChainState;
-  #[wasm_bindgen(typescript_type = "Promise<IdentityState>")]
+  #[wasm_bindgen(typescript_type = "Promise<IdentityState | undefined | null>")]
```

Comment:
This comment still stands: https://github.com/iotaledger/identity.rs/pull/660/files#r809865709

I think we should just use `null`, not `undefined`, for optional return values.

Reply:
The reason for this is that `napi` automatically generates this types when using `Option`

Reply:
Also, `stronghold.d.ts` incorrectly links to the wrong path from the Wasm bindings (probably why the source mapping fails). Is this a problem for publishing?

```
import { DID, KeyLocation, ChainState, IdentityState, Storage } from "../../wasm/node/identity_wasm";
```

Aside: do we need the `index.ts` file? It also has relative links that are incorrect.

Reply:
turns out we needed a whole build step to resolve this. Went with rollup and a couple plugins to properly bundle the sources. Also added the dependency in the `@iota/identity-wasm` package, which requires linking because storage is not released yet.